src/issue/issue.action.js

```diff
@@ -117,6 +121,56 @@ export const postIssueComment = (body, owner, repoName, issueNum) => {
   };
 };
 
+export const deleteIssueComment = (issueCommentId, owner, repoName) => {
+  return (dispatch, getState) => {
+    const accessToken = getState().auth.accessToken;
+
+    dispatch({ type: DELETE_ISSUE_COMMENT.PENDING });
```

Comment:
I know this is not in the scope of this PR but what do you think about using `redux-actions` instead? We can reduce boilerplate significantly.

You can find and example in this PR #187 

If you're not feeling ok with it, that's fine too!

Reply:
Oh, that's pretty amazing, I really like this approach! But to be honest, I think this should be done in a separated PR, I think is better just refactor the whole issue.action.js instead only this particular part, just for keeping the cohesion of the file.

What do you think about? 😄

Reply:
Completely agree!

Thanks!!!